app/sequencescape_excel/sequencescape_excel/column.rb

```diff
@@ -91,6 +91,7 @@ def attribute_value(detail)
     end
 
     def specialised_field?
+      # We can't use const_defined? here as we want to make sure we trigger rails class loading
```

Comment:
I broke things by trying to be clever. Exception handling isn't the fastest in Ruby, and const_defined? is ten times faster than rescuing the exception. However it fails to trigger the Rails class loading behaviour, so will never trigger loading of the specialised field classes in test and development. I added this as a warning to either my future self, or other moments of hubris.